tests/unit/ml/test_compose.py

```diff
@@ -173,3 +179,355 @@ def test_columntransformer_repr_matches_sklearn():
     )
 
     assert bf_column_transformer.__repr__() == sk_column_transformer.__repr__()
+
```

Comment:
Thanks for adding unit tests. Can you also add system tests to verify that 
1. .fit() and .transform() work as expected in standalone transformer
2. .fit() and .predict() work as expected in pipeline
3. to_gbq() and the returned(loaded) transformer works as expected, if supported.

You can find examples in tests/system/